charts/shoot-core/charts/coredns/values.yaml

```diff
@@ -0,0 +1,74 @@
+# Default values for coredns.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+
+# The values here match the ones on the upstream deployment repo (https://github.com/coredns/deployment/blob/master/kubernetes/coredns.yaml.sed). 
+# They may need to be updated in the future
+
+service:
+  type: "ClusterIP"
+  clusterDNS: 100.64.0.10 # This can changed if required, execute `kubectl get service --namespace kube-system kube-dns -o jsonpath="{.spec.clusterIP}` to get the clusterDNS IP
+  domain: cluster.local in-addr.arpa ip6.arpa
+  annotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/port: "9153"
+
+images:
+   coredns: image-repository:image-tag
+
+deployment:
+  spec:
+    containers:
+    # image:
+       # coredns: image-repository:image-tag
+      imagePullPolicy: IfNotPresent
+      resources:
+        limits:
+          cpu: 200m
+          memory: 170Mi
+        requests:
+          cpu: 100m
+          memory: 70Mi
+
+configmap:
+  servers:
+  - zones:
+    - zone: .
+    port: 53
```

Comment:
Can we host it on unprivileged port, like `8053`? So we can drop `NET_BIND_SERVICE` capability.

Reply:
Hi @mvladev, 

Are you scrapping Kube-dns ? I am asking because I only found the job referenced in the rules, but nothing in the [config](https://github.com/gardener/gardener/blob/master/charts/seed-monitoring/charts/prometheus/templates/config.yaml)

Reply:
There is a template which uses the K8S service discovery in Prometheus (with the `prometheus.io/...` annotations) - https://github.com/gardener/gardener/blob/174ea1122aef8f39b37106fc2496b173be5fd04b/charts/seed-monitoring/charts/prometheus/templates/_helpers.tpl#L1-L29
It is being used in the Prometheus configuration:

https://github.com/gardener/gardener/blob/174ea1122aef8f39b37106fc2496b173be5fd04b/charts/seed-monitoring/charts/prometheus/templates/config.yaml#L232-L247

Reply:
I see, thanks!